hummingbird/ml/operator_converters/__init__.py

```diff
@@ -18,8 +18,9 @@
 
 # To register a converter for scikit-learn API operators, import associated modules here.
 from . import lightgbm  # noqa: E402
-from . import onnx_tree_ensembles  # noqa: E402
+from . import onnx_linear_ensembles  # noqa: E402
```

Comment:
Can we call this file just `onnxml_linear`? (There are no linear ensembles here)